### ISSUE TITLE
Windows support broken on pypi

### ISSUE THREAD

**Issue by Happyzippy** [cirospaciari/socketify.py#87]
**Describe the bug**
Problem with linking to libsocketify_windows_amd64.dll

**To Reproduce**
1. Run web sockets example
```from socketify import App, AppOptions, OpCode, CompressOptions

def ws_open(ws):
    print('A WebSocket got connected!')
    ws.send("Hello World!", OpCode.TEXT)

def ws_message(ws, message, opcode):
    #Ok is false if backpressure was built up, wait for drain
    ok = ws.send(message, opcode)
    
app = App()    
app.ws("/*", {
    'compression': CompressOptions.SHARED_COMPRESSOR,
    'max_payload_length': 16 * 1024 * 1024,
    'idle_timeout': 12,
    'open': ws_open,
    'message': ws_message,
    'drain': lambda ws: print(f'WebSocket backpressure: {ws.get_buffered_amount()}'),
    'close': lambda ws, code, message: print('WebSocket closed'),
    'subscription': lambda ws, topic, subscriptions, subscriptions_before: print(f'subscription/unsubscription on topic {topic} {subscriptions} {subscriptions_before}'),
})
app.any("/", lambda res,req: res.end("Nothing to see here!'"))
app.listen(3000, lambda config: print("Listening on port½ http://localhost:%d now\n" % (config.port)))
app.run()
```

**Result:**
```
Exception ignored from cffi callback <function uws_generic_listen_handler at 0x000001C93EB06F80>:
Traceback (most recent call last):
  File "c:\Users\emtyg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\Local\pypoetry\Cache\virtualenvs\postr-ZLbm3cEi-py3.10\lib\site-packages\socketify\socketify.py", line 1016, in uws_generic_listen_handler
    config.port = lib.us_socket_local_port(app.SSL, listen_socket)
  File "c:\Users\emtyg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\Local\pypoetry\Cache\virtualenvs\postr-ZLbm3cEi-py3.10\lib\site-packages\cffi\api.py", line 912, in __getattr__      
    make_accessor(name)
  File "c:\Users\emtyg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\Local\pypoetry\Cache\virtualenvs\postr-ZLbm3cEi-py3.10\lib\site-packages\cffi\api.py", line 908, in make_accessor    
    accessors[name](name)
  File "c:\Users\emtyg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\Local\pypoetry\Cache\virtualenvs\postr-ZLbm3cEi-py3.10\lib\site-packages\cffi\api.py", line 838, in accessor_function
    value = backendlib.load_function(BType, name)
AttributeError: function/symbol 'us_socket_local_port' not found in library 'c:\Users\emtyg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\Local\pypoetry\Cache\virtualenvs\postr-ZLbm3cEi-py3.10\lib\site-packages\socketify\libsocketify_windows_amd64.dll': error 0x7f
```

**Desktop:**
 - OS: Windows 10
 - Version: python-3.10, socketify-0.0.6 and socketify-0.0.5

**Comment by cirospaciari** [comment]
Thanks for reporting, an update in uSockets library broke this, tomorrow I will release the fix

**Comment by cirospaciari** [comment]
Fixed with version v0.0.7 if you find anything wrong please reopen this issue
